src/main/java/heronarts/lx/midi/surface/MidiFighterTwister.java

```diff
@@ -226,6 +231,13 @@ private void resend() {
       }
     }
 
+    void registerFocusedChannel() {
```

Comment:
Not worth re-uploading here but toss a `private` on there in a future rev

Reply:
I'll change it on the next edit.  It was copied from the APC40.. I'll let you fix that one since you have a list going.